src/components/StoreItemCard/index.tsx

```diff
@@ -1,9 +1,47 @@
 import React from 'react';
+import { ReactComponent as Diamond } from '../../assets/icons/Diamond.svg';
+import CatBread from '../../assets/icons/cat-bread.jpg';
```

Comment:
![image](https://user-images.githubusercontent.com/33337209/101226691-30d9ec00-364a-11eb-871a-f5023fbd5c95.png)

Reply:
@shravanhariharan2 stick to backend 👺